ipv4/set.go

```diff
@@ -316,3 +317,20 @@ func (me Set) Difference(other SetI) Set {
 func (me Set) isValid() bool {
 	return me.trie.isValid()
 }
+
```

Comment:
TODO: This would be a really good time to get the CI running on this repo and display the shiny "passing" README badge.

Reply:
Added here: https://github.com/go-addrs/addrs/pull/16